linux_os/guide/system/software/integrity/software-integrity/aide/aide_disable_silentreports/bash/ubuntu.sh

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_ubuntu
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+{{{ bash_package_install("aide") }}}
```

Comment:
are you sure you need remediation?
the used template should be enough I think.

Reply:
This is not necessary, I'll remove the shell script. Thanks for the feedback!